practices/package_management.md

```diff
@@ -52,38 +64,38 @@ Here are some examples of Package Managers:
 - [apk-tools](http://apk-tools.sourceforge.net/) for UNIX like RAM-based embedded systems
 - [dpkg](https://wiki.debian.org/Teams/Dpkg) for Debian (and now Ubuntu)
 
-## Why You Might Want the Package Management
+## Why You Might Want Package Management
 
 Package Managers:
 
-- simplify work for developers
-- allow you to upgrade the packages very fast, you do not have to upgrade one by one
-- save the time
-- also have some advanced features - for example, they can make a basic audit
-- can maintain the project versions (automatically upload them online) so everyone uses the newest version
-- allow you to manage someone else’s open source code with certainty that the build is consistent within the developers and environments
+- Simplify work for developers
+- Allow you to upgrade the packages very fast, you do not have to upgrade one by one
+- Save time
+- Include some advanced additional features - for example, they can make a basic audit
+- Maintain project versions (automatically upload them online) so that everyone uses the newest version. This greatly assists business support functions when attempting to diagnose the cause of, and eliminate, any issues.
+- Allow you to manage someone else’s open source code with certainty that the build is consistent within the developers and environments
 
 ## Problems the Package Management Solves
 
 - [Increased cost](/problems/increased-cost)
 - [Poor code quality](/problems/poor-code-quality)
 - [Meaningless work](/problems/meaningless-work)
-- [Demotivated team](/problems/demotivated-team)
+- [De-motivated team](/problems/demotivated-team)
```

Comment:
Please change to Demotivated. It is the name of another article so we will keep this form.